pkg/ebpf/c/maps.h

```diff
@@ -114,6 +129,12 @@ BPF_LRU_HASH(bpf_attach_tmp_map, u32, bpf_attach_t, 1024);         // temporaril
 BPF_PERCPU_ARRAY(event_data_map, event_data_t, 1);                 // persist event related data
 BPF_HASH(logs_count, bpf_log_t, bpf_log_count_t, 4096);            // logs count
 BPF_PERCPU_ARRAY(scratch_map, scratch_t, 1);                       // scratch space to avoid allocating stuff on the stack
+BPF_QUEUE(found_hidden_modules_queue, kernel_module_t, 1024);      // modules that were found to be hidden and to be reported to userspace
+BPF_HASH(modules_from_procfs, u64, kernel_module_name_t, 2048);    // modules returned by reading /proc
+BPF_QUEUE(walk_mod_tree_queue, rb_node_t, 2048);                   // used to walk a rb tree
+BPF_HASH(unhidden_modules_map, u64, u64, 2048);                    // modules that are shown (not hidden)
+BPF_LRU_HASH(already_reported_hidden_modules_map, u64, u64, 1024); // the hidden modules that we already reported
```

Comment:
Maps we have today are generic, meaning that they are used by more than one event.
Here you add 5 new maps just for 1 event, which will be created on any invocation of tracee, even if the event was not chosen. Few things to consider:
1. Reduce the amount of used maps
2. Move these maps declaration near to the event that use them
3. We need to find a way to load these maps only when this event is requested by the user